src/io/FileSystem.hx

```diff
@@ -114,6 +116,25 @@ class FileSystem
 		Fs.writeFileSync(path, data);
 	}
 
+	public static function saveRGBAToPNG(data:Uint8Array, width:Int, height:Int, path:String)
```

Comment:
Using js.Lib.require seems hacky but idk how to do another way. Thoughts?

Also overall this is a pretty convoluted RGBA -> PNG conversion but it doesn't require external dependencies.